main.go

```diff
@@ -129,13 +124,13 @@ func doSlaveReport(masterAddr *net.UDPAddr) bool {
 	// it doesn't really matter what port we listen for
 	lAddr, err := net.ResolveUDPAddr("udp", ":0")
 	if err != nil {
-		llog.Fatal("error resolving report listen UDP addr", kv.Set("err", err))
```

Comment:
This should still be a fatal. If we can't get a port to listen on, it should die, something insane happened.